specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1889,6 +2158,109 @@
       ],
       "description": "The List disk encryption set operation response."
```

Comment:
encryption set?

Reply:
this is part of the listDiskEncryptionSet response.. not list DiskAccess